docs/Developers/Git_Basics.rst

```diff
@@ -110,6 +122,15 @@ It will give you an idea about what files are currently modified.
 Adding the files and commiting
 ------------------------------
 
+First, make sure you're on the correct branch and not developing on master! If
```

Comment:
maybe you should talk about ``stash`` in case you have already done the modification on master